doc-site/docs/tutorials/private-storage.md

```diff
@@ -0,0 +1,180 @@
+# Private Storage Contract
+
+In this tutorial, you'll learn how to deploy and interact with a **private storage contract** using **Paladin’s privacy groups**. Unlike the **public storage contract**, where data is visible to everyone, **private storage ensures that only authorized members** of a privacy group can interact with the contract.
+
```

Comment:
Feels like we should link to whatever docs we have on Pente? This tutorial gives you a good step by step to get hands on Pente privacy groups, but does not explain what they actually are. Probably need a pointer to get more info on them.